site2/docs/reference-configuration.md

```diff
@@ -338,8 +338,12 @@ brokerServiceCompactionThresholdInBytes| If the estimated backlog size is greate
-|transactionMetadataStoreProviderClassName|                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |org.apache.pulsar.transaction.coordinator.impl.InMemTransactionMetadataStoreProvider|
+|transactionCoordinatorEnabled| Whether to enable transaction coordinator in broker.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |false|
+|transactionMetadataStoreProviderClassName||org.apache.pulsar.transaction.coordinator.impl.MLTransactionMetadataStoreProvider|
+|transactionBufferSnapshotMaxTransactionCount|Transaction Buffer will take a snapshot after the number of transaction operations reaches this value.|1000|
+|transactionBufferSnapshotMinTimeInMillis|Transaction buffer take snapshot interval time (Unit : millisecond)|5000|
+|transactionPendingAckLogIndexMinLag|MLPendingAckStore maintains a ConcurrentSkipListMap pendingAckLogIndex. It stores the position in pendingAckStore as its value and saves a position used to determine whether the previous data can be cleaned up as a key. transactionPendingAckLogIndexMinLag is used to configure the minimum lag between indexes|500|
```

Comment:
who `stores the position in pendingAckStore as its value and saves a position ....`? can you specify it explicitly?